Ninja-star/NinjaStar/GamePlayScene.swift

```diff
@@ -0,0 +1,122 @@
+//
+//  GameScene.swift
+//  NinjaStar
+//
+//  Created by Nissan Adid on 3/03/18.
+//  Copyright © 2018 Nissan Adid. All rights reserved.
+//
+
+import SpriteKit
+import GameplayKit
+
+class GamePlayScene: SKScene {
+   
+   private var label : SKLabelNode?
+   private var spinnyNode : SKShapeNode?
```

Comment:
You can delete these if your not using them, they just come with the generated project.